api/AuthenticatorAssertionResponse.json

```diff
@@ -4,19 +4,24 @@
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/AuthenticatorAssertionResponse",
         "support": {
-          "chrome": {
-            "version_added": "65",
-            "notes": "Only support USB U2F tokens",
-            "flags": [
-              {
-                "type": "preference",
-                "value_to_set": "true",
-                "name": "Web Authentication API"
-              }
-            ]
-          },
+          "chrome": [
+            {
+              "version_added": "67"
+            },
+            {
+              "version_added": "65",
+              "notes": "Only support USB U2F tokens",
+              "flags": [
+                {
+                  "type": "preference",
+                  "value_to_set": "Enabled",
+                  "name": "Web Authentication API"
+                }
+              ]
```

Comment:
I don't know much about how other vendors use flags. For Chrome the expectation is that after something is available by default that the flags should be used EVER. Full stop. I don't see the value of keeping the history.

Reply:
The current policy is to keep such information, but I can see your point that it isn't very useful after it is enabled by default. Maybe something to talk to @Elchi3 about when get gets back from Xmas vacation.

Reply:
If there is no benefit for data consumers, I'm fine with removing it. It looks like the time span the flag was there is really small, so it will be totally irrelevant in like a year anyways.

Reply:
To be honest, I've been trying to come up with an argument that's better than 'Google doesn't like it', to counter the arguments for keeping it. The two I've heard are that MDN should treat browsers consistently and that 'broadly available' includes the flag stage. In our view, a flagged feature is not part of the web platform. It's part of Chrome's implementation process. We don't support flagged features beyond experimentation on a desktop and discourage it when asked. If a web developer wants to experiment with a feature and it's enabled by default, then it's the production version they should be experimenting with.